cmd/crowdsec-cli/decisions.go

```diff
@@ -0,0 +1,304 @@
+	"github.com/spf13/cobra"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var Scope, Value, Type, DecisionID string
+var DeleteAll bool
+var Client *apiclient.ApiClient
+
+func AlertsToTable(alerts *models.GetAlertsResponse) error {
+	if csConfig.Cscli.Output == "raw" {
+		fmt.Printf("id,source,ip,reason,action,country,as,events_count,expiration\n")
+		for _, alertItem := range *alerts {
+			for _, decisionItem := range alertItem.Decisions {
+				fmt.Printf("%v,%v,%v,%v,%v,%v,%v,%v,%v\n",
+					decisionItem.ID,
+					*decisionItem.Origin,
+					*decisionItem.Scope+":"+*decisionItem.Target,
+					*decisionItem.Scenario,
+					*decisionItem.Type,
+					alertItem.Source.Cn,
+					alertItem.Source.AsNumber+" "+alertItem.Source.AsName,
+					*alertItem.EventsCount,
+					*decisionItem.Duration)
+			}
+		}
+	} else if csConfig.Cscli.Output == "json" {
+		x, _ := json.MarshalIndent(alerts, "", " ")
+		fmt.Printf("%s", string(x))
+	} else if csConfig.Cscli.Output == "human" {
+
+		table := tablewriter.NewWriter(os.Stdout)
+		table.SetHeader([]string{"ID", "Source", "Target", "Reason", "Action", "Country", "AS", "Events", "Expiration"})
+
+		if len(*alerts) == 0 {
+			fmt.Println("No active decisions")
+			return nil
+		}
+
+		for _, alertItem := range *alerts {
+			for _, decisionItem := range alertItem.Decisions {
+				table.Append([]string{
+					strconv.Itoa(int(decisionItem.ID)),
+					*decisionItem.Origin,
+					*decisionItem.Scope + ":" + *decisionItem.Target,
+					*decisionItem.Scenario,
+					*decisionItem.Type,
+					alertItem.Source.Cn,
+					alertItem.Source.AsNumber + " " + alertItem.Source.AsName,
+					strconv.Itoa(int(*alertItem.EventsCount)),
+					*decisionItem.Duration,
+				})
+			}
+		}
+		table.Render() // Send output
+	}
+	return nil
+}
+
+func NewDecisionsCmd() *cobra.Command {
+	/* ---- DECISIONS COMMAND */
+	var cmdDecisions = &cobra.Command{
+		Use:   "decisions [action]",
+		Short: "Manage decisions",
+		Long: `
+Decisions Management.
+
+To list/add/delete decisions
+`,
+		Example: `cscli decisions [action] [filter]`,
+		Args:    cobra.MinimumNArgs(1),
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			var err error
+			if csConfig.LapiClient == nil {
+				log.Fatalln("There is no configuration on 'api_client:'")
+			}
+			apiclient.BaseURL, err = url.Parse(csConfig.LapiClient.Credentials.Url)
+			if err != nil {
+				log.Fatalf("failed to parse Local API URL %s : %v ", csConfig.LapiClient.Credentials.Url, err.Error())
+			}
+
+			password := strfmt.Password(csConfig.LapiClient.Credentials.Password)
+			t := &apiclient.JWTTransport{
+				MachineID: &csConfig.LapiClient.Credentials.Login,
+				Password:  &password,
+				Scenarios: []string{"aaaaaa", "bbbbb"},
+			}
+
+			Client = apiclient.NewClient(t.Client())
+		},
+	}
+
+	var cmdDecisionsList = &cobra.Command{
+		Use:     "list --scope [scope] --value [value] --type [type]",
+		Short:   "List decisions",
+		Long:    `List decisions from the LAPI`,
```

Comment:
maybe we want something more user focused like "List active decisions" (for the help). If 90% of the users are running single machine setup, we might want to "hide" the complexity of the local API service ?